src/index.test.ts

```diff
@@ -44,6 +45,32 @@ describe('commitlint', () => {
           );
         });
       });
+
+      describe('with custom messageReplacer function', () => {
```

Comment:
We'll want one more test for when the function is `undefined` or of the wrong type.

Reply:
For the `undefined` one,  I thought it's already covered here in the default config part (or would you prefer a more explicit describe block for it?)
https://github.com/BearAlliance/danger-plugin-conventional-commitlint/blob/6a0cf3eda4177ff78529aa3ce0d13c3e35270a53/src/index.test.ts#L39-L41

As for the wrong type, isn't the typescript warning enough? 
![image](https://user-images.githubusercontent.com/42243887/213304766-f09ef9ac-5b6b-41bd-acb0-3c90e2d37405.png)

Let me know if I misunderstood or you had something else in mind!

Reply:
> For the undefined one, I thought it's already covered here in the default config part

Yeah good point, I was only looking at the additions 😅 

> As for the wrong type, isn't the typescript warning enough?

It is, so long as the consumer of the plugin is using Type Script 😉 
Internal to the module, we can trust the type checking, but the interface with the user should be slightly more robust. Right now they'd get the default `TypeError`, which I guess you could argue is probably fine.